recipe/meta.yaml

```
@@ -1,40 +1,47 @@
-{% set version = "4.5.1" %}
+{% set name = "podman-py" %}
+{% set version = "5.4.0.1" %}
 
 package:
-  name: podman-py
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/containers/podman-py/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 464631a8005e61a5118c2d408f373a327efb854074b70ef6e24985d90c0b2231
+  url: https://pypi.org/packages/source/{{ name[0] }}/podman/podman-{{ version }}.tar.gz
```

Please remove all templating of the name, an replace it with its actual value.
```suggestion
  url: https://pypi.org/packages/source/p/podman/podman-{{ version }}.tar.gz
```